### ISSUE TITLE
Bug in createComment mutation, commentOn field

### ISSUE THREAD

**Issue by romcik** [wp-graphql/wp-graphql#1975]
Hi, I wanted report bug in createComment mutation.
If you set input field "commentOn" to non existing post_id, you will get error: "Cannot use object of type UserError as array. "

It's caused here
https://github.com/wp-graphql/wp-graphql/blob/develop/src/Mutation/CommentCreate.php
line 132

```
if ( empty( $commented_on ) ) {
return new UserError( __( 'The ID of the node to comment on is invalid', 'wp-graphql' ) );
}
```
There should be "throw" instead "return"
```
if ( empty( $commented_on ) ) {
throw new UserError(__('The ID of the node to comment on is invalid', 'wp-graphql'));
}
```

Than graphql will return right message: "The ID of node to comment on is invalid"

**Comment by bhardie** [comment]
This issue is being discussed in #1952.  Please follow along the discussion there.  This issue will be closed as a duplicate.

**Comment by romcik** [comment]
@bhardie U checked thread you mentioned and it looks it's different topic, than I'm referring to.

**Comment by justlevine** [comment]
@romcik this is indeed a separate bug. I've addressed it in #2323
